# HG changeset patch
# User Michael Tharp <michael.tharp@sas.com>
# Date 1374587744 0
# Node ID 757a858b22a0c7676ad0d37780e829466f9abe7b
# Parent  67d0874bd3344dfede9da1cc8a938f55ffb7fd9b
cfg: add 'writeDoc' method to OptionValue for backwards compatibility with rbuild (RCE-1979)

diff --git a/conary/lib/cfg.py b/conary/lib/cfg.py
--- a/conary/lib/cfg.py
+++ b/conary/lib/cfg.py
@@ -252,6 +252,9 @@
         self.updateFromString(newStr, path, line)
         return self.value
 
+    def writeDoc(self, out, displayOptions=None):
+        return self.definition.writeDoc(out, displayOptions)
+
 
 def directive(func):
     """
